bigbluebutton-html5/imports/ui/components/presentation/presentation-overlay/component.jsx

```diff
@@ -1,7 +1,11 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import SlideCalcUtil from '/imports/utils/slideCalcUtils';
 
 const CURSOR_INTERVAL = 16;
+const MYSTERY_NUM = 2;
```

Comment:
Please add some inline comment for this mystery number... What happens if it's not 2?